doc/source/programs/gdal_raster_index.rst

```diff
@@ -0,0 +1,154 @@
+.. _gdal_raster_index_subcommand:
+
+================================================================================
+"gdal raster index" sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Create a vector index of raser datasets.
```

Comment:
Create a vector index of raster datasets.

Reply:
fixed